database/src/main/scala/cromwell/database/slick/WorkflowStoreSlickDatabase.scala

```diff
@@ -24,7 +24,7 @@ trait WorkflowStoreSlickDatabase extends WorkflowStoreSqlDatabase {
                                      (implicit ec: ExecutionContext):
   Future[Seq[WorkflowStoreEntry]] = {
 
-    val action = for {
+    val action: DBIO[Seq[WorkflowStoreEntry]] = for {
```

Comment:
Yeah.  Also we didn't intend to make changes to the workflow store, we just happened to pick this as the basis for a sanity check.  😄